Rick-and-Morty/Rick And Morty.xcodeproj/project.pbxproj

```diff
@@ -408,6 +411,7 @@
 				B811688A1CFF1C9900301A0A /* Release */,
 			);
 			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
```

Comment:
can you revert this? It will use release as default configuration while we should use debug

Reply:
I think it went automatically when I did the swift migration. Will revert

Reply:
@gbasile I can't find how to change this